includes/taxonomy.php

```diff
@@ -2,7 +2,7 @@
 require_once(ABSPATH . 'wp-admin/includes/taxonomy.php');
 
 function research_briefings_wp_get_or_create_category() {
-	$category_name = 'Briefing paper';
+	$category_name = 'Research Briefing';
```

Comment:
Whilst this is a good quick-fix, it does mean all previous Briefing papers will be categorised, still, as 'Briefing paper'. I'd prefer to flesh this out a little to replace previous category groupings with the new category name. What're your thoughts @olihawkins?

Reply:
Richard Keen made this change. He seems to think the older papers have been reclassified correctly. Can you drop him an email to resolve whether this is the case? Let me know how it affects any changes I should make to the PR.

Reply:
As with above's comment, it will have done (as it would have updated the previous Briefing papers). Not to worry about this one.

Reply:
Great. Thanks Jake. I will amend the number of briefings being pulled each hour and resubmit the pull request soon.